Delete manifests/t/Tableau/Desktop/2023.3.0 directory

dobsontom opened the pull request:
- [x] Have you signed the [Contributor License Agreement](https://cla.opensource.microsoft.com/microsoft/winget-pkgs)?
- [x] Have you checked that there aren't other open [pull requests](https://github.com/microsoft/winget-pkgs/pulls) for the same manifest update/change?
- [x] This PR only modifies one (1) manifest
- [x] Have you [validated](https://github.com/microsoft/winget-pkgs/blob/master/AUTHORING_MANIFESTS.md#validation) your manifest locally with `winget validate --manifest <path>`?
- [x] Have you tested your manifest locally with `winget install --manifest <path>`?
- [x] Does your manifest conform to the [1.5 schema](https://github.com/microsoft/winget-pkgs/tree/master/doc/manifest/schema/1.5.0)?

Note: `<path>` is the name of the directory containing the manifest you're submitting.

---

 ###### Microsoft Reviewers: [Open in CodeFlow](https://microsoft.github.io/open-pr/?codeflow=https://github.com/microsoft/winget-pkgs/pull/127095)

dobsontom committed:
Delete manifests/t/Tableau/Desktop/2023.3.0 directory

dobsontxx commented:
Deleting version 2023.3.0 to keep with convention of other versions (should be 23.3.345).

dobsontom committed:
Delete manifests/t/Tableau/TableauPrepBuilder/2023.3.0 directory

Deleting version 2023.3.0 to keep with convention of other versions (should be 23.3.30378).
